sdk/storage/azblob/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 0.3.0 (Unreleased)
 
 ### Features Added
+* Exported fields `Offset` and `Count` on `HttpRange`
```

Comment:
This is not exported, but is accessed as if it is exported in one of the examples.

Reply:
I've fixed this change in my other PR. So we should be able to export it from next release onwards.